.github/workflows/release.yml

```diff
@@ -114,6 +115,10 @@ jobs:
             provider: aws
             rack_params: "node_type=t4g.small proxy_protocol=true"
             suffix: arm64
+          - job: aws-downgrade
```

Comment:
Did you check about the manual approval for this job?
You can implement it in a different ticket if you want, let's just track it in the milestone.

Reply:
Created another ticket for that: https://github.com/convox/issues-private/issues/606